shell/index.html

```diff
@@ -10,60 +10,54 @@
 
         <!-- Injection point for base URL from backend -->
         <meta name="dhis2-base-url" content="__DHIS2_BASE_URL__" />
+
         <!--
             manifest.json provides metadata used when your web app is installed on a
             user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
         -->
         <link
             rel="manifest"
             crossorigin="use-credentials"
-            href="%PUBLIC_URL%/manifest.json"
+            href="/manifest.json"
```

Comment:
Vite resolves these URLs relative the the `base` path specified in the Vite config